src/stories/Ory/Auth.stories.tsx

```diff
@@ -87,6 +88,20 @@ LoginAuthCardPasswordless.args = {
   },
 }
 
+export const LoginAuthCardRefresh = Template.bind({})
+
+LoginAuthCardRefresh.args = {
+  title: "Two-Factor Authentication",
```

Comment:
Wouldn't the title here be "Confirm it is you"

Reply:
To be changed also in kratos-selfservice.